modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -457,12 +455,12 @@ public TableManager(
         stableAssignmentsRebalanceListener = createStableAssignmentsRebalanceListener();
 
         assignmentsSwitchRebalanceListener = createAssignmentsSwitchRebalanceListener();
+
+        mvGc = new MvGc(nodeName, tablesCfg.gcThreads().value());
```

Comment:
Please never read configuration in the constructor of the component, it's not yet initialized.

Reply:
Fix it